src/scenes/ProgressReports/Detail/ProgressReportDrawer/Steps/PromptVariant/PromptsForm.tsx

```diff
@@ -0,0 +1,81 @@
+import { Box, Typography } from '@mui/material';
+import { SubmissionErrors } from 'final-form';
+import { Form } from 'react-final-form';
+import { EnumField, EnumOption, SubmitButton } from '~/components/form';
+import { required } from '~/components/form/validators';
+import { RichTextView } from '~/components/RichText';
+import { ProgressReportAvailableDataFragment } from '../../ProgressReportDrawer.graphql';
+
+interface PromptsFormProps {
+  availableData: ProgressReportAvailableDataFragment | null;
+  reportId: string;
+  createItemMutation: (
+    input: any
+  ) => void | SubmissionErrors | Promise<SubmissionErrors>;
+}
+
+export const PromptsForm = ({
+  availableData: stepData,
+  reportId,
+  createItemMutation,
+}: PromptsFormProps) => {
+  return (
+    <Form
+      onSubmit={createItemMutation}
+      initialValues={{
+        reportId,
```

Comment:
I typically just grab this from scope. I know this is how it's done in traditional html/server-side forms. I just don't see the need here. This probably makes sense for the current way you are connecting the submit handler. It'll probably be less needed once looking at my other comments.
[An example](https://github.com/SeedCompany/cord-field/blob/2667b2d8c09f382454516be3ac56579195262df9/src/scenes/Products/Edit/EditProduct.tsx#L171-L174).